CommandLineClient/src/main/java/com/dslplatform/compiler/client/parameters/DslCompiler.java

```diff
@@ -797,4 +797,36 @@ public String getDetailedDescription() {
 				"\tcompiler=/var/dsl-platform/dsl-compiler.exe\n" +
 				"\tcompiler=12345\n";
 	}
+
+	private static final String[] MONO_NATIVE_ERROR_SNIPPETS = {
+			"SIGSEGV",
+			"Native stacktrace:",
+			"mono() [0x",
+			"Debug info from gdb:",
+			"Invalid IL code"
+	};
+
+	/**
+	 * Checks if the output contains one of the regexes which could indicate
+	 * a recurring mono bug.
+	 */
+	private static boolean monoNativeFailure(String output) {
+		if (output == null) {
+			return false;
+		} else {
+			for (String snippet : MONO_NATIVE_ERROR_SNIPPETS) {
+				if (output.contains(snippet)) return true;
```

Comment:
I meant to check for \nmono [0x instead of just mono [0x
It's nothing major... but I kind of don't like we are doing it on normal output... so if someone puts in a comment about how crappy mono is, not to break ;(